pg_shard.c

```diff
@@ -258,21 +280,42 @@ PgShardPlanner(Query *query, int cursorOptions, ParamListInfo boundParams)
 		selectFromMultipleShards = SelectFromMultipleShards(query, queryShardList);
 		if (selectFromMultipleShards)
 		{
-			Oid distributedTableId = InvalidOid;
```

Comment:
Just assign the actual value here if we're going to need it in this scope (and I think we will?)